script/demoScripts/demoGarden.ts

```diff
@@ -0,0 +1,341 @@
+          WETH_ADDRESS,
+          wallet,
+          LIFI_ADDRESS,
+          inputAmount,
+          false
+        )
+      }
+
+      // Step 1: Get quote from Garden API
+      consola.info('\n📡 Fetching quote from Garden API...')
+
+      const quoteUrl = `https://api.garden.finance/v2/quote?from=ethereum:usdc&to=base:usdc&from_amount=${amountInWei.toString()}`
+      const response = await fetch(quoteUrl)
+
+      if (!response.ok)
+        throw new Error(`Failed to fetch quote: ${response.statusText}`)
+
+      const quote: IGardenQuote = await response.json()
+
+      if (quote.status !== 'Ok' || !quote.result || quote.result.length === 0)
+        throw new Error('No valid quote received from Garden API')
+
+      const selectedQuote = quote.result[0]
+      if (!selectedQuote) throw new Error('No quote selected')
+
+      consola.success(`Quote received:`)
+      consola.info(
+        `  Input: ${selectedQuote.source.display} ${selectedQuote.source.asset
+          .split(':')[1]
+          ?.toUpperCase()}`
+      )
+      consola.info(
+        `  Output: ${
+          selectedQuote.destination.display
+        } ${selectedQuote.destination.asset.split(':')[1]?.toUpperCase()}`
+      )
+      consola.info(`  Solver: ${selectedQuote.solver_id}`)
+
+      // Step 2: Submit order to Garden API
+      consola.info('\n📝 Submitting order to Garden...')
+
+      const orderPayload = {
+        source: {
+          asset: 'ethereum:usdc',
+          owner: address,
+          amount: amountInWei.toString(),
+        },
+        destination: {
+          asset: 'base:usdc',
+          owner: address, // Same address on destination chain
+          amount: selectedQuote.destination.amount,
+        },
+      }
+
+      const orderResponse = await fetch(
+        'https://api.garden.finance/v2/orders',
+        {
+          method: 'POST',
+          headers: {
+            'garden-app-id':
+              '7648702b1997e55a3763afa5dd7ace3d4bd23348ee0423cc27a18ef3e28cb2b7', // [pre-commit-checker: not a secret]
+            'Content-Type': 'application/json',
+          },
+          body: JSON.stringify(orderPayload),
+        }
+      )
+
+      if (!orderResponse.ok) {
+        const errorText = await orderResponse.text()
+        throw new Error(
+          `Failed to submit order: ${orderResponse.statusText} - ${errorText}`
+        )
+      }
+
+      const orderData: IGardenOrderResponse = await orderResponse.json()
+
+      if (orderData.status !== 'Ok' || !orderData.result)
+        throw new Error('Failed to create order with Garden')
+
+      consola.success(`Order created: ${orderData.result.order_id}`)
+      consola.info(
+        `  Redeemer: ${orderData.result.typed_data.message.redeemer}`
+      )
+      consola.info(
+        `  Timelock: ${orderData.result.typed_data.message.timelock}`
+      )
+      consola.info(
+        `  Secret Hash: ${orderData.result.typed_data.message.secretHash}`
```

Comment:
I currently do not yet understand how the user (or we) get access to the actual secret value. 
Also I dont find any information at all about this facet in this page: 
https://www.notion.so/lifi/Third-Party-Integrations-Bridges-etc-b71470e096d5481ba7dbca4ba555cd58